__tests__/app.test.js

```diff
@@ -14,21 +14,54 @@ afterAll(() => {
 
 describe('GET /api/topics', () =>{
     test('status:200, should return an array of topic objects, with slug and description properties', () => {
-        //Arrange
         return request(app)
-        //Act
         .get('/api/topics')
-        //Assert
         .expect(200)
         .then(({body}) => {
             const { topics } = body
                 expect(topics.length).toBe(3)
             topics.forEach((topic) => {
-                expect(topic).toHaveProperty('slug')
-                expect(topic).toHaveProperty('description')
-                expect(typeof topic.slug).toBe('string')
-                expect(typeof topic.description).toBe('string')
+                expect(topic).toHaveProperty('slug', expect.any(String))
+                expect(topic).toHaveProperty('description', expect.any(String))
             })
         })
     })
+})
+
+describe('GET /api/articles/:article_id', () =>{
+    test('status:200, should return an article object, with correct properties when the endpoint is a valid article ID number', () => {
+        return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({body}) => {
+            const { article } = body
+                expect(article.article_id).toBe(1)
+                expect(article).toMatchObject({
+                    article_id: 1,
+                    title: 'Living in the shadow of a great man',
+                    topic: 'mitch',
+                    author: 'butter_bridge',
+                    body: 'I find this existence challenging',
+                    created_at: '2020-07-09T20:11:00.000Z',
```

Comment:
This is fine! If you want this to be a number or date you could manipulate in the query/model or you can deal with this in your front end!